From 111d99e10a401b999fb1c65bfc000a13bd88b7bd Mon Sep 17 00:00:00 2001
From: George Mocanu <george.mocanu@nxp.com>
Date: Mon, 17 Jun 2024 13:29:42 +0300
Subject: [PATCH] Remove ipc-shm-xen module

ipc-shm-xen and ipc-shm-dev modules export the same symbols,
affecting the dependency used within the GoldVIP IPC char driver.

Signed-off-by: George Mocanu <george.mocanu@nxp.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 1dcae0d..4dab526 100644
--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,7 @@ EXTRA_CFLAGS ?=
 ifneq ($(KERNELRELEASE),)
 # kbuild part of makefile
 
-obj-m := $(MODULE_NAME).o $(CDEV_MODULE_NAME).o $(UIO_MODULE_NAME).o $(XEN_MODULE_NAME).o
+obj-m := $(MODULE_NAME).o $(CDEV_MODULE_NAME).o $(UIO_MODULE_NAME).o
 
 $(MODULE_NAME)-y := ipc-shm.o ipc-queue.o os_kernel/ipc-os.o
 $(XEN_MODULE_NAME)-y := ipc-shm.o ipc-queue.o os_kernel/ipc-xen.o
-- 
2.25.1

